cmd/ingest.go

```diff
@@ -65,9 +68,17 @@ func (c *ingestCmd) Command() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:               "ingest",
-		Short:             "Run Ingestion service",
-		PersistentPreRunE: utils.DefaultPersistentPreRunE(cfgOpts),
+		Use:   "ingest",
+		Short: "Run Ingestion service",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			utils.DefaultPersistentPreRunE(cfgOpts)
```

Comment:
This won't work, the `utils.DefaultPersistentPreRunE` will only return a function, and not call it. I'd suggest you look at this [example](https://github.com/stellar/wallet-backend/pull/33/files#diff-87123721308af5d4a9f1dac03f5438ba1b3b5adf3e4bbf12666587666953a5d6R53-R78) to see how we implement the `PersistentPreRunE`.

Reply:
Done